Modules/posixmodule.c

```diff
@@ -5195,6 +5195,14 @@ parse_posix_spawn_flags(PyObject *setpgroup, int resetids, PyObject *setsigmask,
         all_flags |= POSIX_SPAWN_RESETIDS;
     }
 
+    if (setsid) {
+#ifdef POSIX_SPAWN_SETSID
+    all_flags |= POSIX_SPAWN_SETSID;
+#elif defined(POSIX_SPAWN_SETSID_NP)
+    all_flags |= POSIX_SPAWN_SETSID_NP;
+#endif
```

Comment:
Wait. An older version of the PR raised an exception using argument_unavailable_error() if the constant was missing, no? The doc says the parameter *requires* a constant.

Please reintroduce argument_unavailable_error() call.